### PR TITLE
Various changes

### PR DESCRIPTION
**Pull request by OzGav**

### CODE REVIEW

**Review by jozefKruszynski — COMMENTED** [review]
As I understand it having more than 1 h1 tag or `#` per page is generally not advised.

File: docs/player-support/snapcast.md [hunk 1]
```diff
@@ -34,13 +38,13 @@ Snapcast packages are available for several Linux distributions:
 - [Archlinux](#archlinux)
 - [Void Linux](#void-linux)
 
-### Debian
+<h3>Debian</h3>
```
Comment (jozefKruszynski) [inline docs/player-support/snapcast.md:41]: These should be the same, but I'd rather stick with the md versions for consistency

**Comment by OzGav** [comment]
Not sure of the reason for that but below that level mkdocs will add it to the TOC. For the FAQ pages with long headings it gets messy in the sidebar...? I can change them all to <h2> in the code which won't get indexed according to what I read

File: docs/player-support/snapcast.md (hunk 1 above)
Comment (OzGav) [reply docs/player-support/snapcast.md:41]: Dont delberately so they dont get indexed as per here https://github.com/squidfunk/mkdocs-material/issues/313#issuecomment-299698639

**Comment by jozefKruszynski** [comment]
> Not sure of the reason for that but below that level mkdocs will add it to the TOC. For the FAQ pages with long headings it gets messy in the sidebar...? I can change them all to <h2> in the code which won't get indexed according to what I read

You can use yaml front matter to override certain things on a per page basis

https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#hiding-the-sidebars

**Comment by OzGav** [comment]
Yeah I don't necessarily want to hide the side bar just some levels of the headings in the TOC. For example on the snapcast page it goes to level 5 which blows out the TOC on the side and then you can't see what else there is as it is off the page and you have to scroll. I think that is messy when I think those headings are inconsequential and can be omitted by using the alternate HTML?

File: docs/player-support/snapcast.md (hunk 1 above)
Comment (jozefKruszynski) [reply docs/player-support/snapcast.md:41]: Fair enough, then that should be the way to go I guess

**Comment by jozefKruszynski** [comment]
> Yeah I don't necessarily want to hide the side bar just some levels of the headings in the TOC. For example on the snapcast page it goes to level 5 which blows out the TOC on the side and then you can't see what else there is as it is off the page and you have to scroll. I think that is messy when I think those headings are inconsequential and can be omitted by using the alternate HTML?

Fair enough, what I find weird is that all the top stuff is supposed to be collapsible, yet so far I haven't found the possibility of how to do that.
Lets stick to the html tags to avoid the indexing

**Comment by OzGav** [comment]
Thanks. If you find a better way I am all for it.

**Review by jozefKruszynski — APPROVED** [review]

**Pull request merged**